src/styles/tools/ec-dropdown.css

```diff
@@ -0,0 +1,13 @@
+@define-mixin ec-dropdown-search-item
+  $item-horizontal-padding: 8px,
+  $item-vertical-padding: 16px {
+  padding: $item-horizontal-padding $item-vertical-padding;
```

Comment:
Do we need to have these as variables? I think it could be just `padding: 8px 16px` or try to use tailwind: `@apply tw-py-16 tw-px-8`

Reply:
The original mixin had this variables, but I agree, we can remove in favor of the utility classes

- [ ] Use class